.github/workflows/build.yml

```diff
@@ -126,7 +126,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest]
```

Comment:
shim can no longer be compiled on macOS?

If so, maybe make a DCGMWrapper interface and provide two implementations: real implementation when go-dcgm works and dummy implementation when it doesn't and pick it at compile time via https://pkg.go.dev/cmd/go#hdr-Build_constraints.

Reply:
Thanks for the idea, I'll bring back macOS support in a separate PR.